shifthelper/checks.py

```diff
@@ -2,6 +2,7 @@
 from . import retry_smart_fact_crawler as sfc
 from .tools.is_shift import is_shift_at_the_moment
 import requests
+from requests.exceptions import RequestException
```

Comment:
why `RequestException` and not `requests.exceptions.ConnectionError`. I tested it with `requests.exceptions.ConnectionError`  and it worked. I guess you tested it with `requests.exceptions.RequestException` and it also worked. So maybe you can explain why you chose this one?